legal-api/requirements.txt

```diff
@@ -8,47 +8,94 @@ Flask==1.1.2
 Jinja2==2.11.3
 Mako==1.1.4
 MarkupSafe==1.1.1
+PyHamcrest==2.0.2
+PyNaCl==1.4.0
+PyYAML==5.4.1
 SQLAlchemy-Continuum==1.3.11
 SQLAlchemy-Utils==0.36.8
-SQLAlchemy==1.3.23
+SQLAlchemy==1.3.24
 Werkzeug==1.0.1
 alembic==1.5.7
 aniso8601==9.0.1
+astroid==2.5.2
 asyncio-nats-client==0.11.4
 asyncio-nats-streaming==0.4.0
 attrs==20.3.0
+autopep8==1.5.6
+bcrypt==3.2.0
 blinker==1.4
 cachelib==0.1.1
 certifi==2020.12.5
+cffi==1.14.5
 chardet==4.0.0
 click==7.1.2
+coverage==5.5
+cryptography==3.4.7
 datedelta==1.3
+distro==1.5.0
+docker-compose==1.29.0
+docker==5.0.0
 dpath==2.0.1
 ecdsa==0.14.1
 expiringdict==1.1.4
+flake8-debugger==4.0.0
+flake8-docstrings==1.6.0
+flake8-isort==4.0.0
+flake8-polyfill==1.0.2
+flake8==3.9.0
 flask-jwt-oidc==0.3.0
-flask-restx==0.2.0
-gunicorn==20.0.4
+flask-restx==0.3.0
+freezegun==1.1.0
+gunicorn==20.1.0
+hypothesis==6.8.9
 idna==2.10
+iniconfig==1.1.1
+isort==5.8.0
 itsdangerous==1.1.0
 jsonschema==3.2.0
 launchdarkly-server-sdk==7.1.0
+lazy-object-proxy==1.6.0
+mccabe==0.6.1
+packaging==20.9
+paramiko==2.7.2
+pep8-naming==0.11.1
+pluggy==0.13.1
 protobuf==3.15.6
 psycopg2-binary==2.8.6
+py==1.10.0
 pyRFC3339==1.1
 pyasn1==0.4.8
+pycodestyle==2.7.0
 pycountry==20.7.3
+pycparser==2.20
+pydocstyle==3.0.0
+pyflakes==2.3.1
+pylint-plugin-utils==0.6
+pylint==2.7.4
+pyparsing==2.4.7
 pyrsistent==0.17.3
+pytest-asyncio==0.14.0
```

Comment:
looks like you included all of the modules in requirements/dev.txt into requirements.txt
Please remove the dev/test related ones.